examples/step-90/doc/results.dox

```diff
@@ -0,0 +1,26 @@
+<h1>Results</h1>
+
+The numerical solution for a very finest mesh is shown below.
+
+@image html step-90-solution.png
+
+<h3>Convergence test </h3>
+
+The results of the convergence study is shown in the table below.
+
+| Cycle |   DOFS   |   EOC | Iterations | $L^2$-Error |   EOC | $H^1$-Error |   EOC |Stabilization|   EOC |
+|:-----:|:--------:|:-----:|:----------:|:-----------:|:-----:|:-----------:|:-----:|:-----------:|:-----:|
+| 0     |    12370 |   -   |   15       | 7.6322e-02  |  -    | 3.6212e-01  |  -    | 2.2423e-01  |    -  |
+| 1     |    49406 | -2.00 |   18       | 1.1950e-02  | 2.68  | 1.4752e-01  | 1.30  | 1.1238e-01  | 1.00  |
+| 2     |   196848 | -1.99 |   19       | 1.7306e-03  | 2.79  | 7.4723e-02  | 0.98  | 6.1131e-02  | 0.88  |
+| 3     |   785351 | -2.00 |   22       | 3.6276e-04  | 2.25  | 3.9329e-02  | 0.93  | 3.0185e-02  | 1.02  |
+| 4     |  3136501 | -2.00 |   25       | 7.5910e-05  | 2.26  | 1.9694e-02  | 1.00  | 1.4875e-02  | 1.02  |
+| 5     | 12536006 | -2.00 |   26       | 1.7279e-05  | 2.14  | 9.8443e-03  | 1.00  | 7.4067e-03  | 1.01  |
+| 6     | 50122218 | -2.00 |   30       | 4.3891e-06  | 1.98  | 4.9219e-03  | 1.00  | 3.7042e-03  | 1.00  |
```

Comment:
What does EOC stand for? And why would you show EOC on the number of DoFs?

Reply:
Ah, I see. Perhaps put that interpretation into text below the table to make it clear what you want the interpretation to be.